content/blog/releases/announcing-opencue-v0.3.33-release.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Announcing the release of OpenCue v0.3.33"
+linkTitle: "v0.3.33 release"
+date: 2020-01-29
+description: "OpenCue v0.3.33 release notes"
+---
+
+This release includes alpha-level support for Windows jobs and hosts. More
+information on Windows support has been posted to the
+[opencue-user mailing list](https://lists.aswf.io/g/opencue-user).
```

Comment:
I think link target here should be https://lists.aswf.io/g/opencue-user/topic/windows_support_alpha/70232740

Reply:
Thanks and updated.